### ISSUE TITLE
How to remove categories from sidebar ?

### ISSUE THREAD

**Issue by leoplct** [google/docsy#888]
How to remove those categories in the sidebar but keep the structure on the right-side?

I haven't found anything about in https://www.docsy.dev/docs/adding-content/navigation/

![Screenshot 2022-02-13 at 13 58 48](https://user-images.githubusercontent.com/1176145/153754271-37ae26bf-c5c0-4b46-8152-c6e920accac6.png)

**Comment by deining** [comment]
To get rid of this section, remove or comment out this line in the base layout for `docs`:

https://github.com/google/docsy/blob/5958d944d2d308b24123cb8b3c85d40ab20aa369/layouts/docs/baseof.html#L19

In order to keep this change when upgrading your theme, you are advised to copy this file `baseof.html` over to the `layouts` folder in your site root:

```
your-site-root/layouts/docs/baseof.html
```

and remove line 19 in the copied file.

**Comment by leoplct** [comment]
Would be cool to have a boolean variable in config.toml so will be easier to upgrade without copy-paste the layouts again.

**Comment by narrenfrei** [comment]
There are already options to control the output:

https://www.docsy.dev/docs/adding-content/taxonomy/#parameters

**Comment by leoplct** [comment]
I had already those options in config.toml but it was still showing until I've removed the line from baseof.html
```
# Comment out to enable taxonomies in Docsy
# disableKinds = ["taxonomy", "taxonomyTerm"]

# You can add your own taxonomies
[taxonomies]
tag = "tags"
category = "categories"

[params.taxonomy]
# set taxonomyCloud = [] to hide taxonomy clouds
taxonomyCloud = [] 

# If used, must have same lang as taxonomyCloud
taxonomyCloudTitle = [] 

# set taxonomyPageHeader = [] to hide taxonomies on the page headers
taxonomyPageHeader = [] 

```

**Comment by Inrmanish** [comment]
@leoplct see my pull req,
gd or not

**Comment by narrenfrei** [comment]
Already found the problem. I'm working on a bugfix.

**Comment by narrenfrei** [comment]
Should be solved by PR #895 

I had to switch from @narrenfrei  to @raum51, which is my new github username.

**Comment by deining** [comment]
> Should be solved by PR #895

I tested this, and yes, if I set in my `config.toml`

```
[params.taxonomy]
  taxonomyCloud = []
```

the entries `Categories` and `Tags` in the sidebar disappear. So this issue can be closed.
